inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -1404,6 +1741,8 @@ void MKLDNNReduceNode::getSupportedDescriptors() {
     if (!descs.empty())
         return;
 
+    setPostOps(attr, true);
```

Comment:
Considering that in the case of dyncamic shapes `attr` can be changed depending on the specific shape, apparently this method should be called on `prepareParams()` stage.

Reply:
Fixed. Thanks Maksim!